recipes/otagrum/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.2" %}
+
+package:
+  name: otagrum
+  version: {{ version }}
+
+source:
+  #fn: otagrum-{{ version }}.tar.gz
+  #url: https://github.com/openturns/otagrum/archive/v{{ version }}.tar.gz
+  #sha256: 853eff2e7ff59a03424c9b03bf9d508c59ccde668cc056a7da312170475bc30c
+
+  # Use git waiting for the 0.2 release
+  git_url: https://github.com/openturns/otagrum.git
+  git_rev: c39134403da3cc88b8d9ec496ee219d3ee10170f
```

Comment:
Hmm We normally don't use git for our sources: https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos

Reply:
I know, a stable release should be coming. I'll use the tarball then, like all my other ot* packages.